rllib/examples/two_trainer_workflow.py

```diff
@@ -0,0 +1,131 @@
+"""Example of using a custom training workflow.
+
+Here we create a number of CartPole agents, some of which are trained with
+DQN, and some of which are trained with PPO. Both are executed concurrently
+via a custom training workflow.
+"""
+
+import argparse
+import gym
+
+import ray
+from ray import tune
+from ray.rllib.agents.trainer_template import build_trainer
+from ray.rllib.agents.dqn.dqn import DEFAULT_CONFIG as DQN_CONFIG
+from ray.rllib.agents.dqn.dqn_tf_policy import DQNTFPolicy
+from ray.rllib.agents.ppo.ppo import DEFAULT_CONFIG as PPO_CONFIG
+from ray.rllib.agents.ppo.ppo_tf_policy import PPOTFPolicy
+from ray.rllib.execution.concurrency_ops import Concurrently
+from ray.rllib.execution.metric_ops import StandardMetricsReporting
+from ray.rllib.execution.rollout_ops import ParallelRollouts, ConcatBatches, \
+    StandardizeFields, SelectExperiences
+from ray.rllib.execution.replay_ops import StoreToReplayBuffer, Replay
+from ray.rllib.execution.train_ops import TrainOneStep, UpdateTargetNetwork
+from ray.rllib.examples.env.multi_agent import MultiAgentCartPole
+from ray.rllib.optimizers.async_replay_optimizer import LocalReplayBuffer
+from ray.tune.registry import register_env
+from ray.util.iter import LocalIterator
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--num-iters", type=int, default=20)
+
+
+def custom_training_workflow(workers, config):
+    local_replay_buffer = LocalReplayBuffer(
+        num_shards=1,
+        learning_starts=1000,
+        buffer_size=50000,
```

Comment:
maybe comment here what the units are (learning_starts= 1000 steps? batches? samples?)

Reply:
same with buffer size